scripts/install/SetOfflineTestRunnerConfig.php

```diff
@@ -46,7 +47,12 @@ class SetOfflineTestRunnerConfig extends \common_ext_action_InstallAction
     public function __invoke($params)
     {
         $this->updateTestRunnerConfig();
-        $this->registerOfflineProxy();
+        if (!$this->registerOfflineProxy()) {
+            return new \common_report_Report(
+                \common_report_Report::TYPE_ERROR,
```

Comment:
[optional] the FQN can be simplified

Reply:
you mean something like `new Report(Report::TYPE_ERROR, ...);` ?